stylix/target.nix

```
@@ -5,6 +5,7 @@ with lib;
 let
   fromOs = import ./fromos.nix { inherit lib args; };
 in {
+  options.stylix.enable = lib.mkEnableOption "stylix" // { default = true; example = false; };
```

> The option defaults to `true`, because that is the current behavior of stylix.

Although backwards compatibility is good, it usually causes more harm than good in the long run. Considering that nearly every `enable` option in the Nix ecosystem defaults to `false`, I suggest introducing a breaking change of defaulting `stylix.enable` to `false`:

```suggestion
  options.stylix.enable = lib.mkEnableOption "stylix";
```

Bear in mind that upgrading Stylix without an explicit `stylix.enable = true;` declaration should result in a fairly obvious error message.